docs/how-to-add-support-for-new-package-manager.md

```diff
@@ -0,0 +1,43 @@
+# How to add support for a new package manager
+
+The sections below outline the steps needed to add support for a new package manager to OSS Review Toolkit.
+
+## 1. Initial Questions / Requirements
+
+Before you start any implementation work, determine whether the package manager meets the minimal requirements to provide the data needed for OSS Review Toolkit to work.
+
+Answers to the questions below should help you 
+
+1. Detect whether a project is a pnpm project 
+
+> The presence of [pnpm-lock.yaml](https://github.com/pnpm/pnpm/blob/master/pnpm-lock.yaml) which is pnpm lockfile or pnpm-workspace.yaml which is used for workspaces.
+
+2. Detect declared license for a package
+
+> pnpm is part of Node.js ecosystem so it has package.json and uses npmjs.com as its central package repository.
+
+3. How to get dependency tree including package names, versions
+
+> https://pnpm.js.org/en/cli/list
+
+4. Obtain the source for a dependency
+
+> Same as npm, pnpm queries registry.npmjs.com
+
+5. Determine how to separate code dependencies from build/test ones
+
+> Same scopes as `npm` and `yarn` - `dependencies` amd `devDependencies` 
+
+6. Example pnpm projects to test implementation
+
+> https://github.com/zkochan/packages/tree/2cb8e0072168869e86d8a81206330352455746cd/rename-overwrite
+
+## 2. Adding new module to the Analyzer
```

Comment:
"Adding a new Analyzer module" to align with headers below.

Reply:
done